src/Microsoft.AspNetCore.SpaTemplates/content/Aurelia-CSharp/Properties/launchSettings.json

```diff
@@ -0,0 +1,41 @@
+{
+  "iisSettings": {
+//#if (WindowsAuth)
+    "windowsAuthentication": true,
+    "anonymousAuthentication": false,
+//#else
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+//#endif
+    "iisExpress": {
+      "applicationUrl": "http://localhost:55556/",
+//#if (RequiresHttps)
+      "sslPort": 43434
+//#else
+      "sslPort": 0
+//#endif
+    }
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+//#if (RequiresHttps)
+      "launchUrl": "https://localhost:43434/",
+//#endif
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+//#if (!WindowsAuth)
+    },
+    "Company.WebApplication1": {
```

Comment:
Where does the HTTPS port config come from for people running directly under Kestrel, not IISExpress?

Reply:
I'll need to update the templates to have ASPNETCORE_URLS, which will parse the HTTPS port.